packages/mdc-snackbar/mdc-snackbar.scss

```diff
@@ -21,152 +21,183 @@
-      left: 0;
-      width: 100%;
-      transform: translate(0);
-    }
-  }
+@at-root {
+  @include mdc-snackbar-fill-color(
+    $mdc-snackbar-overlay-fill-color,
+    $mdc-snackbar-overlay-fill-opacity,
+    $mdc-snackbar-surface-fill-color
+  );
+  @include mdc-snackbar-label-ink-color($mdc-snackbar-label-ink-color, $mdc-snackbar-label-ink-opacity);
+  @include mdc-snackbar-min-width($mdc-snackbar-min-width);
+  @include mdc-snackbar-max-width($mdc-snackbar-max-width);
+  @include mdc-snackbar-elevation($mdc-snackbar-elevation);
+  @include mdc-snackbar-shape-radius($mdc-snackbar-shape-radius);
 }
 
-.mdc-snackbar__action-wrapper {
-  @include mdc-rtl-reflexive-property(padding, 24px, 0);
+.mdc-snackbar--wide {
+  @include mdc-snackbar-viewport-margin($mdc-snackbar-viewport-margin-wide);
 }
 
-.mdc-snackbar--action-on-bottom {
-  flex-direction: column;
+.mdc-snackbar--leading {
+  justify-content: flex-start;
 }
 
-.mdc-snackbar__text {
-  @include mdc-typography(body1);
-  @include mdc-rtl-reflexive-box(margin, right, auto, ".mdc-snackbar");
+.mdc-snackbar--open,
+.mdc-snackbar--closing {
+  transform: none;
+}
 
+.mdc-snackbar__surface {
   display: flex;
+
+  // Non-static positioning is required to make `overflow: hidden` work in Safari.
+  // See https://bugs.webkit.org/show_bug.cgi?id=98538#c5
+  position: relative;
   align-items: center;
-  height: 48px;
-  transition: mdc-animation-exit-permanent(opacity, .3s);
+  justify-content: flex-start;
+  box-sizing: border-box;
+  transform: scale(.8);
   opacity: 0;
-  color: $mdc-snackbar-foreground-color;
 
-  @media (min-width: $mdc-snackbar-tablet-breakpoint) {
-    @include mdc-rtl-reflexive-property(padding, 0, 24px);
+  // Prevent mdc-snackbar__overlay's background-color from bleeding past the surface's border-radius.
+  overflow: hidden;
+
+  .mdc-snackbar--stacked & {
+    flex-direction: column;
+    align-items: flex-start;
+  }
+
+  .mdc-snackbar--open & {
+    transform: scale(1);
+    transition:
+      mdc-animation-enter(opacity, $mdc-snackbar-enter-duration),
+      mdc-animation-enter(transform, $mdc-snackbar-enter-duration);
+    opacity: 1;
+    pointer-events: auto; // Allow mouse events on surface element while snackbar is open
+  }
+
+  .mdc-snackbar--closing & {
+    transform: scale(1);
+    transition: mdc-animation-exit-permanent(opacity, $mdc-snackbar-exit-duration, $mdc-snackbar-exit-delay);
   }
 }
 
-.mdc-snackbar--action-on-bottom .mdc-snackbar__text {
-  margin-right: inherit;
+.mdc-snackbar__overlay {
+  position: absolute;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  box-sizing: border-box;
+  z-index: -1;
 }
 
-.mdc-snackbar--action-on-bottom .mdc-snackbar__action-wrapper {
-  @include mdc-rtl-reflexive-box(margin, left, auto);
+.mdc-snackbar__label {
+  @include mdc-typography($mdc-snackbar-label-type-scale);
 
-  flex-direction: column;
-  justify-content: flex-start;
-  margin-top: -12px;
-  margin-bottom: 8px;
+  display: none;
```

Comment:
Do we still need to set this to `display: none` when the root has `aria-hidden="true"`?

Reply:
Yes, `display: none` is necessary to prevent the elements from being focusable when the user presses the <kbd>Tab</kbd> key.